docs/pages/blog/turbo-1-4-0.mdx

```diff
@@ -0,0 +1,134 @@
+
+- Filesystem monitoring
+- Cache management
+
+### Filesystem monitoring
+
+Right now, the Turborepo Agent has been given just one job: monitor the output locations of tasks. But having filesystem monitoring in place enables additional opportunities with tracking of task inputs to precompute task hashes and perform differential cache restoration.
+
+### Cache management
+
+The last step of any Turborepo build is to store the build outputs with the remote cache. Depending on your network connection this can be either near-instantaneous or prohibitively slow. We plan to hand that job off to the Turborepo Agent so you’ll never have to wait for network calls again.
+
+## Under the hood of Turborepo
+
+Turborepo Agent has been a major effort including decisions about the future architecture of Turborepo.
+
+[![Header of PR which landed the Turborepo Agent.](/images/blog/turbo-1-4-0/pr.png)](https://github.com/vercel/turborepo/pull/1429)
+
+Now, when you start up `turbo`, the first thing we do is launch the Turborepo Agent. We have to wait for that to happen so we use a binary exponential backoff for our connection attempts. This ensures we get out of the way as quickly as possible so you can start building.
+
+That interprocess communication uses gRPC and protobufs to give us versioning guarantees (both backwards- and forwards-compatible) and quick marshaling of data so that we don’t spend all of our time chatting.
+
+Unsupervised processes are wasteful and the majority of the code that was added was to make process handling as robust as possible.
+
+## New examples
+
+Over the past month we’ve invested a lot of time in [creating new examples for how to integrate Turborepo into your workflow](https://github.com/vercel/turborepo/tree/main/examples), featuring:
+
+- Svelte
+- Docker
+- Create React App
+- React Native
+- Prisma
+- Remix
+- … and more!
+
+## Automatic environment variable inclusion
+
+One of the easiest mistakes to make when setting up Turborepo is accidentally omitting an important environment variable resulting in reusing a cache somewhere you didn't expect. Turborepo now has your back and will detect commonly used environment variables to help you avoid this problem.
+
+## Other bug fixes and improvements
+
+- Allow both sides of git comparison. ([#1442](https://github.com/vercel/turborepo/pull/1442))
+- Properly rebuild packages that share a name prefix. ([#1538](https://github.com/vercel/turborepo/pull/1538))
+- Cache files with the correct file permissions. ([#1429](https://github.com/vercel/turborepo/pull/1429))
+
+## Community
+
+Since releasing [Turborepo v1.3 in June](/blog/turbo-1-3-0), we've seen incredible adoption and community growth:
+
+- [8.5k+ GitHub Stars](https://github.com/vercel/turborepo)
+- as many as 325k weekly NPM downloads
+- 8.1 years of compute time saved through Remote Caching on Vercel, saving 8 months per week
```

Comment:
It's actually 9.23 as of today. 10months last week

Reply:
This is scoped to remote caching so 8.1 is the correct number. https://vercel.com/solutions/turborepo